### DISCUSSION TITLE
init awesome colorscheme theme on nvim startup

### DISCUSSION THREAD

**Discussion by wingedseraph** [ibhagwan/fzf-lua#1158] [Q&A] [answered]
if there is any option to autorun (or some command to do it manually) the colorscheme downloaded from awesome_colorscheme in fzf-lua picker when nvim starts? and if there are multiple colorschemes downloaded, how do you autorun the right one?

**Comment by ibhagwan** [comment, marked as answer]
There is no such option, I’m personally not a big fan of automatically downloading stuff from the internet without user interaction and it only takes one key press to download what you want.

In general, awesome colorschemes was created as an explorer to help choose the right color scheme but I have no intention on it being a download/plugin manager, once you found what you want it should be handled by the plugin manager (I.e. lazy.nvim).
